src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,180 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> first;
+    private Node<T> last;
+
+    private static class Node<T> {
```

Comment:
According to the checklist, `Node<T>` should be an inner class and it should be at the end of the class. Please move the `Node<T>` class to the end of `MyLinkedList` class.